### DISCUSSION TITLE
[Unstable] Some weapons shooting backwards sometimes

### DISCUSSION THREAD

**Discussion by HawkinsThe71st** [FakeFishGames/Barotrauma#14001] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

While using the scrap cannon I noticed it would shoot backwards every now and then. Very inconsistent to reproduce but it might occur at specific angles more frequently.
![image](https://github.com/FakeFishGames/Barotrauma/assets/115734437/d8a74730-7d44-4a5d-b9e2-d0d868a0ae00)
Can also happen when using the RFA:
![image](https://github.com/FakeFishGames/Barotrauma/assets/115734437/1a176e95-5c9d-4152-8fab-59263bf93473)

### Reproduction steps

1. Shoot Scrap Cannon or RFA in random directions until the issue occurs

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.5.0.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested the fix for this.
No issues found.
Closing discussion as resolved.
